### ISSUE TITLE
Image quality issues

### ISSUE THREAD

**Issue by kubernetes-bad** [kijai/ComfyUI-ELLA-wrapper#3]
I am having some serious issues with ELLA, would appreciate any advice.

The model in use: https://civitai.com/models/33208?modelVersionId=367245

I'm getting super-toasted images no matter what settings I use.

Here's an example:
sd1.5 prompt: `1girl, catgirl, real cat ears, purple eyes, long eyelashes, round face, cherry lips, pink bows on cat ears, golden corset`
t5 prompt: `Ella is a young catgirl with long lashes, beautiful purple eyes and pretty round face. She wears cherry lip gloss, her feline ears are adorned with pink bows and ribbons. Her ample breasts are covered by golden shiny bra and her panties are golden too. Her cat tail is very long and fluffy, with gorgeous red fur.`

Here's the baseline generated with sd1.5:
![SD1.5 native loader image](https://github.com/kijai/ComfyUI-ELLA-wrapper/assets/153952949/9e0e6665-ce94-47fd-81ec-740378314a2b)

And here is the one generated with ELLA adapter:
![ELLA](https://github.com/kijai/ComfyUI-ELLA-wrapper/assets/153952949/9b8a2359-e16a-4357-a442-52eb018a358a)

What is going on?

**Comment by kijai** [comment]
I don't really know yet myself, seems to work very differently with different schedulers, I'd suggest just trying them all as the generation is pretty fast. Along with trying different cfg values. Photon model was best so far from what I tried.

**Comment by chrish-slingshot** [comment]
I think this partly has to do with not being able to specify a negative prompt. If I prompt a normal KSampler with just a positive prompt and no negative, I get similarly bad quality results.

**Comment by kijai** [comment]
Found one mistake and it should be better now with latest push.

**Comment by kubernetes-bad** [comment]
Alright, this update made it slightly less bad, but still quite blurry and toasted.
Here is the image from same checkpoint loaded natively:
![ComfyUI_temp_dbgyz_00011_](https://github.com/kijai/ComfyUI-ELLA-wrapper/assets/153952949/e1289ace-08bb-4e5f-8aeb-1ff6c7b60406)

and here's the one with latest ELLA-wrapper:
![ComfyUI_temp_khmey_00011_](https://github.com/kijai/ComfyUI-ELLA-wrapper/assets/153952949/662e1ea2-970d-44a1-bda6-9f880d53eba0)

As you can see, there's quite a bit of blurriness going on, details are jagged like compression artifacts.

**Comment by kijai** [comment]
I added euler_a which could help with that, generally I'm not getting that bad results quality wise:
![image](https://github.com/kijai/ComfyUI-ELLA-wrapper/assets/40791699/b791630d-6215-4d8a-8cd6-1de67592f43d)
normal euler:

![image](https://github.com/kijai/ComfyUI-ELLA-wrapper/assets/40791699/b4ff54f5-8045-40b4-a6b9-bbf17aee23f7)
and their default:

![image](https://github.com/kijai/ComfyUI-ELLA-wrapper/assets/40791699/13dedd72-17f1-47fa-a788-29c06357cd50)

**Comment by chrish-slingshot** [comment]
It's definitely improved now, but I think passing in a negative prompt is going to make a huge difference. Below is the same seed/prompt for a normal model with no negative, the same model with a negative prompt, and the ELLA model (4 images each):

![image](https://github.com/kijai/ComfyUI-ELLA-wrapper/assets/117188274/aa1eb394-389f-4df5-a5bd-c37c6ca79635)

You can see the top and bottom are pretty similar in terms of quality, whereas the middle one using the negative prompt is noticeably better.

(By the way, the prompt adherence with ELLA is INSANE! The prompt for these images was "a man with short green hair, red eyes, blue and black striped trousers, and a white t-shirt with a logo of a triangle on it. he has yellow-tinted sunglasses on his head and is holding an red and white umbrella". It nailed it in every single image!)

**Comment by kijai** [comment]
From what I heard from some people it seems it might conflict with models that are trained with booru tags (most anime models). 

There's also native nodes now, which can get much  higher image quality as it works with most other comfy things, but personally I'm struggling to get as good prompt adherence out of it stil:

https://github.com/ExponentialML/ComfyUI_ELLA

**Comment by LuLmaster69** [comment]
> From what I heard from some people it seems it might conflict with models that are trained with booru tags (most anime models).
> 
> There's also native nodes now, which can get much higher image quality as it works with most other comfy things, but personally I'm struggling to get as good prompt adherence out of it stil:
> 
> https://github.com/ExponentialML/ComfyUI_ELLA

Yes, I am also struggling with ExponentialML's implementation as it seems to miss something.

Please continue supporting your repo, as it currently the only reliable way to make ELLA work on ComfyUI

**Comment by jokero3answer** [comment]
> 现在肯定有所改善，但我认为传递负面提示将会产生巨大的影响。下面是没有负面提示的正常模型、有负面提示的相同模型以及 ELLA 模型的相同种子/提示（每个模型 4 个图像）：
> 
> ![图像](https://private-user-images.githubusercontent.com/117188274/321175358-aa1eb394-389f-4df5-a5bd-c37c6ca79635.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMzI5NjcsIm5iZiI6MTcxMzMzMjY2NywicGF0aCI6Ii8xMTcxODgyNzQvMzIxMTc1MzU4LWFhMWViMzk0LTM4OWYtNGRmNS1hNWJkLWMzN2M2Y2E3OTYzNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQxN1QwNTQ0MjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNzhiY2IxNDVkMmNjZGZkZTI4YjNlODZmNjExNjBhYzc2ZDlmYWEwMGE5YjJlMDVjNjg1MWY1ODZiOTMxZWE0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ygVKUycMv9v6_RYR9_yEEK-IHOmn0fPAgYr-0A662Pc)
> 
> 您可以看到顶部和底部的质量非常相似，而中间使用否定提示的质量明显更好。
> 
> （顺便说一句，对 ELLA 的迅速追随太疯狂了！这些图像的提示是“一个绿色短发、红色眼睛、蓝黑条纹长裤、白色 T 恤，上面有三角形标志的男人”头上戴着黄色墨镜，手里拿着一把红白相间的雨伞。”每一张照片都体现了这一点！）

Which model did you use? I've used a couple of controls that didn't work so well.
